src/sentry/tasks/store.py

```diff
@@ -209,34 +209,49 @@ def _do_symbolicate_event(cache_key, start_time, event_id, symbolicate_task, dat
 
     event_id = data["event_id"]
 
+    def _continue_to_process_event():
+        process_task = process_event_from_reprocessing if from_reprocessing else process_event
+        _do_process_event(
+            cache_key=cache_key,
+            start_time=start_time,
+            event_id=event_id,
+            process_task=process_task,
+            data=data,
+            data_has_changed=has_changed,
+            from_symbolicate=True,
+        )
+
+    symbolication_function = get_symbolication_function(data)
+    symbolication_function_name = getattr(symbolication_function, "__name__", "none")
+
     if killswitch_matches_context(
         "store.load-shed-symbolicate-event-projects",
         {
             "project_id": project_id,
             "event_id": event_id,
             "platform": data.get("platform") or "null",
+            "symbolication_function": symbolication_function_name,
         },
     ):
+        _continue_to_process_event()
```

Comment:
perhaps we can make it `return _continue_to_process_event()` (here and below), to indicate that it shouldn't return